src/Tabs/Tab.js

```diff
@@ -62,6 +62,10 @@ export default class Tab extends Component {
      */
     className: PropTypes.string,
     /**
+     * Disables the tab if set to true.
```

Comment:
Please could you follow [this pattern](https://github.com/callemall/material-ui/wiki/Documentation#description-wording) for prop naming?